common/source/docs/common-siyi-rc.rst

```diff
@@ -0,0 +1,36 @@
+.. _common-siyi-rc:
+[copywiki destination="plane,copter,rover,blimp"]
+====================================
+SIYI RC/HD Video Transmission System
+====================================
+
+.. image:: ../../../images/siyi-mk32.png
+    :target: ../_images/siyi-mk32.png
+
+SIYI MK32 is an integrated remote controller, ground station and wireless digital transmission system which can be used with Ardupilot. Up to 15km range transmission of HD Video, control and telemetry is possible, which can be doubled via the use of the SIYI HM30 in repeater mode.
+
+.. image:: ../../../images/siyi-hm30.png
+    :target: ../_images/siyi-hm30.png
+
+Features
+========
+* Android based smart device with 7 inch 1080P screen.
```

Comment:
Do these bullet points appear correctly?  I thought we needed an empty line between the "=="and the "*"

Reply:
![image](https://github.com/user-attachments/assets/b97f18db-37a3-4d41-b014-384597145d4c)
nope